spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/convert/ConverterAwareMappingSpannerEntityWriter.java

```diff
@@ -339,6 +341,16 @@ private static boolean attemptSetIterableValue(Iterable<Object> value,
 		return true;
 	}
 
+
+	private static Value jsonToValueConverter(Object value) {
+		if (value == null) {
+			return Value.json(null);
+		}
+		Gson gson = new Gson();
```

Comment:
This might have been discussed already, but I believe Spring has built-in conversion for JSON where it uses Jackson. Have you looked into trying to use that? Any cons to that?

Reply:
Opting to Gson just because it's already brought in through client libraries, no need to add new dependency.